code/project.py

```diff
@@ -59,12 +80,12 @@ def query_db(sql: str):
     st.write('Date range selected:', start_date, end_date)
 
 sql_sales_range = f"""
-    SELECT S.name as seller, SUM(P.price) as sum
```

Comment:
Let's avoid refactoring things like this unless it is meaningful.

Reply:
keep them like I edited

Reply:
it's meaningful as it describes the data better than just "sum"